src/app/js/formats/istex/Component.js

```diff
@@ -4,7 +4,7 @@ import { List } from 'material-ui/List';
 import translate from 'redux-polyglot/translate';
 import compose from 'recompose/compose';
 
-import FetchPaginatedDataHOC from '../../lib/FetchPaginatedDataHOC';
+import fetchPaginatedDataForComponent from '../../lib/fetchPaginatedDataForComponent';
```

Comment:
Because it is in lib and I want to differentiate from other function or component